src/frontend/screens/Game/GamePage/index.tsx

```diff
@@ -313,11 +320,11 @@ export default React.memo(function GamePage(): JSX.Element | null {
               <div className="infoWrapper">
                 <div className="developer">{developer}</div>
                 <div className="summary">
-                  {extra && extra.about
-                    ? extra.about.description
-                      ? extra.about.description
-                      : extra.about.longDescription
-                      ? extra.about.longDescription
+                  {extraInfo && extraInfo.about
```

Comment:
Maybe put this into a variable to make it cleaner:
`const description = extra?.about?.description || extra?.about?.longDescription || '';`